### PR TITLE
Create CheckBox / Radio

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/components/Checkbox/Checkbox.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        font-size: 12px;
+        margin-right: 8px;
+      }
+
+      &-text {
+        font-size: var(--c-checkbox--font-size);
+      }
+    }
+  }
+</style>
```
Comment (javipuche) [inline src/components/Checkbox/Checkbox.vue]: Convert px to em

File: src/components/Radio/Radio.vue [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<template>
+    <AtCheckbox
+        v-bind="$props"
+        class="c-checkbox--radio"
+        :type="type"
+        value="ass"
```
Comment (javipuche) [inline src/components/Radio/Radio.vue]: Remove

File: src/mixins/formControl.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ export default {
         },
         checked: {
             type: Boolean,
-            default: false
+            default: true
```
Comment (javipuche) [inline src/mixins/formControl.js]: Set false

File: docs/pages/components/Radio.vue [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<template>
+    <Layout>
+        <Markdown>
+            # Radio
+        </Markdown>
+
+        <CodePreview width="435" vertical>
+            <AtRadio label="Radio dfsdf" name="radio-option" />
```
Comment (jjlmoya) [inline docs/pages/components/Radio.vue]: dfsdf

File: src/components/Checkbox/Checkbox.vue [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+            }
+        }
+    }
+</script>
+
+<style lang="scss">
+  .c-checkbox {
+    --c-checkbox-dimensions: 1em;
+    --c-checkbox-font-size: 1em;
+    --c-checkbox-input-border: 1px solid #dfe0e4;
```
Comment (javipuche) [inline src/components/Checkbox/Checkbox.vue]: Convert px to em

**Review by jjlmoya — APPROVED** [review]

File: src/components/Checkbox/Checkbox.vue [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    --c-checkbox-input-background: #fff;
+    --c-checkbox-input-radius: var(--radius-s);
+    --c-checkbox-input-checked-background: #000;
+    --c-checkbox-input-checked-border: 1px solid #000;
```
Comment (javipuche) [inline src/components/Checkbox/Checkbox.vue]: Convert px to em

File: src/components/Checkbox/Checkbox.vue [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+
+    &__input {
+      position: absolute;
+      opacity: 0;
+
+      &:checked {
+        & + label {
```
Comment (javipuche) [inline src/components/Checkbox/Checkbox.vue]: Selectors by class, don't html tags

**Review by javipuche — APPROVED** [review]

**Pull request merged**